packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorNeutralForeground1,
+  colorNeutralForegroundDisabled,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  colorTransparentBackground,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontSizeBase500,
+  fontWeightRegular,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+  lineHeightBase500,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+  spacingHorizontalS,
+} from '../../theme/design-tokens.js';
+
+export const styles = css`
+  /* --- Default styles --- */
+
+  :host {
+    max-width: fit-content;
+  }
+
+  :host .heading {
+    height: 44px;
+    display: grid;
+    position: relative;
+    vertical-align: middle;
+    padding-left: ${spacingHorizontalMNudge};
+    padding-right: ${spacingHorizontalM};
+    border-radius: ${borderRadiusMedium};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+    grid-template-columns: 28px 28px auto 28px;
+  }
```

Comment:
For all selectors in this file, do we need the specificity of `:host` as a prefix? If there's not an attribute we're conditionally styling -such as `:host([my-attribute])`, I don't think we need that at all, it's just extra selectors to process.